tests/integration/test_schema_read.py

```diff
@@ -45,23 +45,12 @@ def test_schema_data_consistency(table, datadir, columns, get_table_data_cols):
     assert not df.empty
 
     sqobj = get_sqobject(table)()
```

Comment:
This sqobj is initialized with the default config file. If it's not present, like in git tests, this sqobject will fail.

``` suggestion
    config_file = create_dummy_config_file()
    sqobj = get_sqobject(table)(config_file=config_file)
```